helloworld-observ-lib/dashboards.libsonnet

```diff
@@ -0,0 +1,133 @@
+local g = import './g.libsonnet';
+local logslib = import 'github.com/grafana/jsonnet-libs/logs-lib/logs/main.libsonnet';
+{
+  local root = self,
+  new(this):
+    local prefix = this.config.dashboardNamePrefix;
+    local links = this.grafana.links;
+    local tags = this.config.dashboardTags;
+    local uid = g.util.string.slugify(this.config.uid);
+    local vars = this.grafana.variables;
+    local annotations = this.grafana.annotations;
+    local refresh = this.config.dashboardRefresh;
+    local period = this.config.dashboardPeriod;
+    local timezone = this.config.dashboardTimezone;
+    local panels = this.grafana.panels;
+    local stat = g.panel.stat;
+    {
+      fleet:
+        g.dashboard.new(prefix + 'Helloworld fleet')
+        + g.dashboard.withPanels(
+          g.util.grid.wrapPanels(
+            [
+              g.panel.row.new('Row 1'),
+              panels.overviewTable1 { gridPos+: { w: 24, h: 16 } },
+              panels.timeSeriesPanel1 { gridPos+: { w: 24 } },
+              panels.timeSeriesPanel1 { gridPos+: { w: 24 } },
+              panels.timeSeriesPanel1 { gridPos+: { w: 12 } },
+              panels.timeSeriesPanel1 { gridPos+: { w: 12 } },
+              panels.timeSeriesPanel1 { gridPos+: { w: 24 } },
+            ], 12, 7
```

Comment:
Minor nitpick, but could you add a comment here to explain what these values mean in this context? 

Might be good to be over the top on comments for the template/hello-world example (and also I simply am not sure 😅 )

Reply:
added inline comment about this.